Makefile

```diff
@@ -112,8 +112,11 @@ clean:
 
 # Clean for all configurations and targets.
 cleanall:
-	rm -rf $(FPGA_DIR)/generated-src
-	rm -rf $(FPGA_DIR)/build
-	rm $(EMULATOR_BIN)
+	rm -rf $(FPGA_DIR)/generated-src; \
```

Comment:
Does it not work to run them on separate lines?

Reply:
Thanks for the suggestion. The original intent was to prevent `make` from stopping on error (when a file it wants to delete does not exist). Using `-f` achieves the same effect.